### PR TITLE
Update wasm deps

### PR DESCRIPTION
**Pull request by grod220**
references https://github.com/penumbra-zone/web/issues/1481

### CODE REVIEW

File: packages/wasm/crate/Cargo.toml [hunk 1]
```diff
@@ -14,39 +14,39 @@ default = ["console_error_panic_hook"]
...
+penumbra-stake = { git = "https://github.com/penumbra-zone/penumbra.git", tag = "v0.79.3", package = "penumbra-stake", default-features = false }
+penumbra-tct = { git = "https://github.com/penumbra-zone/penumbra.git", tag = "v0.79.3", package = "penumbra-tct" }
+penumbra-transaction = { git = "https://github.com/penumbra-zone/penumbra.git", tag = "v0.79.3", package = "penumbra-transaction", default-features = false }
 
 anyhow = "1.0.86"
 ark-ff = { version = "0.4.2", features = ["std"] }
 base64 = "0.22.1"
 console_error_panic_hook = { version = "0.1.7", optional = true }
-decaf377 = { version = "0.5.0", features = ["r1cs"] }
+decaf377 = { version = "0.10.1", features = ["r1cs"] }
```
Comment (TalDerei) [inline packages/wasm/crate/Cargo.toml:37]: how the heck is this building? https://github.com/penumbra-zone/web/pull/1492

Comment (grod220) [inline packages/wasm/crate/Cargo.toml:37]: This seems to just work ™️

Comment (TalDerei) [reply packages/wasm/crate/Cargo.toml:37]: can you hold off on merging? need to sanity check what's going on here

Comment (grod220) [reply packages/wasm/crate/Cargo.toml:37]: They must have changed something in v0.79.3

Comment (TalDerei) [reply packages/wasm/crate/Cargo.toml:37]: I explicitly suggested _not_ including https://github.com/penumbra-zone/penumbra/pull/3806 in the release (which I don't think they did), and https://github.com/penumbra-zone/decaf377/pull/108 which was supposed to fix this hasn't been merged yet. hmmm

**Pull request closed without merging**